main.js

```diff
@@ -36,7 +36,26 @@ function reverseInt(num) {
 /* 4. Return a string with the first letter of every word capitalized
  * ex. capitalizeWords('i love javascript') === 'I Love Javascript'
  */
-function capitalizeWords(str) {}
+function capitalizeWords(str) {
+  let ans = "",
+    j = 0;
+  let len = str.length;
+  for (let i = 0; i < len; i++) {
+    if (j === 0) {
+      ans = ans + str[i].toUpperCase();
```

Comment:
try using higher-order methods like `slice` and `splice` on strings and arrays

Reply:
Done